CONTRIBUTING.md

```diff
@@ -20,6 +20,16 @@ again.
 The Python part of the project will follow [Google Python style guide](http://google.github.io/styleguide/pyguide.html). We provide a [yapf](https://github.com/google/yapf) configuration file to help contributors auto-format their code to adopt the Google Python style. Also, it is encouraged to lint python docstrings by [docformatter](https://github.com/myint/docformatter).
 
 The frontend part of the project uses [prettier](https://prettier.io/) for formatting, read [frontend/README.md#code-style](frontend/README.md#code-style) for more details.
+## Unit Testing Best Practices
+
+* Testing via Public APIs
+
+### Golang
```

Comment:
Golang is completely in `./backend` folder, what do you think about moving the section to `./backend/README.md`.
My personal experience is that, anything local to code is the most discoverable.

Reply:
It looks like all code style issues will be referenced in CONTRIBUTING.md no matter whether it's python code or go code. I think for the code style doc, it should be ok we put it in CONTRIBUTING.md